packages/dev/docs/src/types.js

```diff
@@ -303,16 +303,16 @@ export function InterfaceType({description, properties: props, showRequired, sho
         <table className={`${tableStyles['spectrum-Table']} ${tableStyles['spectrum-Table--quiet']} ${styles.propTable}`}>
           <thead>
             <tr>
-              <td className={tableStyles['spectrum-Table-headCell']}>Name</td>
-              <td className={tableStyles['spectrum-Table-headCell']} style={{'width': '30%'}}>Type</td>
-              {showDefault && <td className={tableStyles['spectrum-Table-headCell']}>Default</td>}
-              <td className={tableStyles['spectrum-Table-headCell']} style={{'width': '40%'}}>Description</td>
+              <td role="columnheader" className={tableStyles['spectrum-Table-headCell']}>Name</td>
```

Comment:
Is `<th>` equivalent?

Reply:
We could use <th>, but then I’d have to change cell alignment and background colors.